Cargo.toml

```diff
@@ -36,7 +35,7 @@ serde_json = { version = "1.0.95" }
 default = ["serde", "k256"]
 ed25519 = ["ed25519-dalek"]
 rust-secp256k1 = ["secp256k1"]
-serde = ["dep:serde", "dep:serde-hex"]
```

Comment:
this dep feature is probably no longer necessary

Reply:
thanks, removed it!